web/scripts/stripe/stripe.js

```diff
@@ -0,0 +1,61 @@
+/*
+sk key : sk_test_tjoHHaJQvf1Y4CoAEpWBWr0w
+publish key :pk_test_4VicAuG5Ou9zuqA4LVIRI0dC
+ from stripe.com
+
+*/
+// Refered to the video https://www.youtube.com/watch?v=FhBtM70QZKM
+// https://stripe.com/docs/stripe-js/elements/quickstart
+// npm install stripe
+// installing body parser
+// $ npm install body-parser
+// API
+
+// npm install express
+
+// install hbs
+// npm install hbs
+
+var express = require('express')
+var stripe = require('stripe')('sk_test_tjoHHaJQvf1Y4CoAEpWBWr0w')
+var bodyParser = require('body-parser')
+var hbs = require('hbs')
```

Comment:
After deleting the views folder, hbs will no longer be used so you can remove this.